### ISSUE TITLE
Create an Installation script to automatically install and setup the project

### ISSUE THREAD

**Issue by xprilion** [anitab-org/mentorship-backend#223]
**Is your feature request related to a problem? Please describe.**
Currently the installation and setup of Mentorship Backend is done manually. It would be great to have a single script which could download the code and setup the various configurations required automatically. 

**Describe the solution you'd like**
Create a shell script named **install.sh** (for Linux/Mac) or **install.bat** (for Windows) which would do the following when run - 
    - Clone the repository to the present working directory of the running script (if not already inside one, which will mostly be the case )
    - Perform all the required setup for the repository to run. Take any variable inputs as mandatory command line arguments, without which the script will produce an error, and prompt the user to enter the values. 
    - Perform unit testing on the installation and display its results

**Describe alternatives you've considered**
To modify the deploy.sh file to work on local systems

**Comment by sara-02** [comment]
@isabelcosta There is a GCI task associated with this, maybe we can tag it as GCI

**Comment by isabelcosta** [comment]
Sounds good @sara-02 and @xprilion ! I will label this as "Program: GCI".

**Comment by sara-02** [comment]
Reference PR: https://github.com/systers/mentorship-backend/pull/203

**Comment by daksha-aeer** [comment]
Working on it!

**Comment by sara-02** [comment]
Just copy pasting my comments from the PR https://github.com/systers/mentorship-backend/pull/267/ for future work on this script, which requires some discussion
For future I am just adding a comment, how can we further improve the installation process if some parts of the script fail. Like if even one of the env fails, the script exits. But I cannot run the script again without first deleting the mentorship-backend folder.
Or how can we detach the python run.py command so that the script can successfully exist once the service is up and running?
Should the script stop if one of the dependency fails to install, how to check that? Or the `tests fail`

**Comment by xprilion** [comment]
@sara-02 my way of proving a good installation was to run the tests at the end of the installation procedure. It is possible to check the exit status of commands in bash, and display errors accordingly.

**Comment by sara-02** [comment]
Yes. I just added the comments to that anyone working on it now or in the future knows the gaps that need to be filled.

**Comment by vj-codes** [comment]
@isabelcosta @devkapilbansal do we need this still ?
